frameworks/elastic/universe/resource.json

```diff
@@ -6,8 +6,8 @@
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
       "scheduler-zip": "{{artifact-dir}}/elastic-scheduler.zip",
       "elasticsearch-jre-tar-gz": "https://downloads.mesosphere.com/elastic/assets/openjdk-{{elasticsearch-openjdk-version}}_linux-x64_bin.tar.gz",
-      "elasticsearch-tar-gz": "https://downloads.mesosphere.com/elastic/assets/elasticsearch-{{elastic-version}}-linux-x86_64.tar.gz",
-      "diagnostics-zip": "https://downloads.mesosphere.com/elastic/assets/support-diagnostics-{{support-diagnostics-version}}-dist.zip",
+      "elasticsearch-tar-gz": "https://infinity-artifacts.s3-us-west-2.amazonaws.com/elastic/elasticsearch-7.4.1-linux-x86_64.tar.gz",
+      "diagnostics-zip": "https://infinity-artifacts.s3-us-west-2.amazonaws.com/elastic/support-diagnostics-7.1.1-dist.zip",
```

Comment:
FYI Murilo and team I suggested Hitesh use the `infinity-artifacts` paths for now in the PR and revert back to `downloads.m` once all test are green